payloads/data-reporting/v3.4/reports-api/responses/chats/ratings.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
I got the following response:

`{
    "name": "ratings-report",
    "request": {
        "distribution": "day",
        "timezone": "Europe/Warsaw",
        "filters": {
            "from": "2021-06-23T02:00:00+02:00",
            "to": "2021-06-28T01:59:59+02:00"
        }
    },
    "total": 0,
    "records": {
        "2021-06-23": {},
        "2021-06-24": {},
        "2021-06-25": {},
        "2021-06-26": {},
        "2021-06-27": {},
        "2021-06-28": {}
    }
}`

Reply:
My chats were not rated, so nevermind, but I also got the `request` object

Reply:
Same for `/ratings`